docs/source/reference/queries.md

```diff
@@ -35,6 +35,32 @@ used in searches like
 c.search(Key("color") == "red").search(Key("temperature") > 300)
 ```
 
+````{warning}
+
+To compound searches, use repeated calls to `.search()` as in
```

Comment:
Should this note aboutcompounding be outside of the warning?

Reply:
Done.